Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -87,15 +89,117 @@ public static void HandleBanlist(Session session, params string[] parameters)
+                }
+
+                switch (selectorType)
+                {
+                    case AccountLookupType.Account:
+                        {
+                            playerSession = WorldManager.Find(bootName);
+                            if (playerSession != null)
+                                bootId = playerSession.Player.Guid.Low;
+                            break;
+                        }
+                    case AccountLookupType.Character:
+                        {
+                            playerSession = WorldManager.FindByPlayerName(bootName);
+                            if (playerSession != null)
+                                bootId = playerSession.Player.Guid.Low;
+                            break;
+                        }
+                    case AccountLookupType.Iid:
+                        {
+                            // Extract the Id from the parameters
+                            uint.TryParse(parameters[1], out bootId);
+                            playerSession = WorldManager.Find(new ObjectGuid(bootId));
+                            if (playerSession != null)
+                                bootName = playerSession.Player.Name;
+                            break;
+                        }
+                }
+
+                if (playerSession != null)
+                {
+                    string bootText = $"account or player: {bootName} id: {bootId}";
+                    // Boot the player
+                    playerSession.BootPlayer();
+
+                    // Send an update to the admin, but prevent sending too if the admin was the player being booted
+                    if (session != null)
+                    {
+                        // Log the player who initiated the boot
+                        bootText = "Player: " + session.Player.Name + " has booted " + bootText;
+                        if (session != playerSession)
+                            session.Network.EnqueueSend(new GameMessageSystemChat(bootText, ChatMessageType.Broadcast));
+                    }
+                    else
+                    {
+                        bootText = "Console booted " + bootText;
+                    }
+
+                    // log the boot to file
+                    ILog log = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+                    log.Info(bootText);
+
+                    // finish execution of command logic
+                    return;
+                }
+            } 
+
+            // Did not find a player
+            string errorText = $"Error locating the player or account to boot.";
+            // Send the error to a player or the console
+            if (session != null)
+            {
+                session.Network.EnqueueSend(new GameMessageSystemChat(errorText, ChatMessageType.Broadcast));
+            } else
+            {
+                Console.WriteLine(errorText);
+            }
```

Comment:
single line if-else need no brackets, that will also fix spacing issue with else